NEWS.md

```diff
@@ -1,5 +1,13 @@
 ## Matrix Depot Release Notes
 
+v0.7.0
```

Comment:
```suggestion
v0.8.0
```
0.7 is the current release so I guess this should be probably be 0.8.

Reply:
Understood - I will push a new commit with "v0.8.0" in NEWS.
Can you mange to release this version then?

Reply:
Yes. I'll make the release once this has been merged.